src/FNA3D_Driver_Vulkan.c

```diff
@@ -92,7 +92,9 @@ static const uint32_t deviceExtensionCount = SDL_arraysize(deviceExtensionNames)
 #define COMMAND_LIMIT 100
 #define STARTING_ALLOCATION_SIZE 64000000 /* 64MB */
 #define MAX_ALLOCATION_SIZE 256000000 /* 256MB */
-#define TEXTURE_STAGING_SIZE 8000000 /* 8MB */
+#define STARTING_TEXTURE_STAGING_SIZE 16000000 /* 16MB */
+#define MAX_FAST_TEXTURE_STAGING_SIZE 128000000 /* 128MB */
```

Comment:
Similar to what we ran into with the last iteration, 128MB is too large and SOR4 takes a dive when Skytrain is loaded.

```suggestion
#define MAX_FAST_TEXTURE_STAGING_SIZE 64000000 /* 64MB */
```